module/VuFind/src/VuFind/Controller/AbstractRecord.php

```diff
@@ -955,8 +957,12 @@ protected function showTab($tab, $ajax = false)
         $view->backgroundTabs = $this->getBackgroundTabs();
         $view->tabsExtraScripts = $this->getTabsExtraScripts($view->tabs);
         $view->loadInitialTabWithAjax
-            = isset($config->Site->loadInitialTabWithAjax)
-            ? (bool)$config->Site->loadInitialTabWithAjax : false;
+            = isset($config->Site->loadInitialTabWithAjax) && (bool)$config->Site->loadInitialTabWithAjax;
+        try {
+            $factory = $this->getService(SearchOriginFactory::class);
+            $this->layout()->setVariable('searchOrigin', $factory->createObject($this->params()->fromQuery()));
+        } catch (Exception) {
```

Comment:
Is there a particular reason for this try..catch? Do you expect errors here? If so, should we at least log problems, even if we don't want a search origin glitch to break the whole request?

Reply:
I duplicated the try catch (in the factory + in the controller), I removed the duplication and added a log